Makefile

```diff
@@ -5,6 +5,9 @@ include pkg/project/testdata/env
 generate-ci: clean generate-eventing-ci generate-serving-ci
 .PHONY: generate-ci
 
+generate-ci-no-clean: generate-eventing-ci generate-serving-ci
+.PHONY: generate-ci-no-clean
```

Comment:
no clean?

Reply:
The other one call `clean`, this one doesn't, hence "no clean"

Reply:
A better name? `without-clean` seems very long

Reply:
no clean is good, after this extra context.
thax